cli/app/app.go

```diff
@@ -593,17 +597,22 @@ func ProjectKuberConvert(p *project.Project, c *cli.Context) {
 		logrus.Fatalf("Failed to parse the compose project from %s: %v", composeFile, err)
 	}
 
-	f := createOutFile(outFile)
-	defer f.Close()
+	var f *os.File
+	if !createChart {
+		f = createOutFile(outFile)
```

Comment:
Do this because we're copying rc files to chart templates now. Can support single file later.